basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/BaseLTIService.java

```diff
@@ -192,6 +194,25 @@ public void destroy() {
 		log.info("destroy()");
 	}
 
+	protected Optional<String[]> getContentModelDaoIfConfigurable(Map<String, Object> tool, boolean isAdminRole) {
```

Comment:
just a little confused here on the use of Optional, why not just always return an Array even if its empty this would avoid the need for Optional?